### PR TITLE
Trace connections on exception

### PR DESCRIPTION
**Pull request by jrs65**
This adds a config option for toggling the tracing of connections when they encounter an exception.

### CODE REVIEW

File: coco/request_forwarder.py [hunk 1]
```diff
@@ -142,6 +207,7 @@ def __init__(self, blocklist_path: os.PathLike, timeout: int):
         self.queue_len = None
         self.queue_wait_time = None
         self.response_time = None
+        self._debug = debug
```
Comment (andrerenard) [inline coco/request_forwarder.py:210]: Should we name this `debug_connections` to be consistent with the config option?  I realize this is inside the forwarder, but in theory `debug` could still mean different things in this context.

**Review by andrerenard — APPROVED** [review]
This all looks good to me, just one optional comment on variable naming.

File: coco/request_forwarder.py (hunk 1 above)
Comment (jrs65) [reply coco/request_forwarder.py]: Good point. Done.

**Comment by jrs65** [comment]
There is an annoying issue that `aioredis` has now moved to a v2 with a fairly different API. @aelanman ported comet to it (though we haven't deployed it at CHIME), but my attempts to port coco to use it haven't been panning out. I do what I think should work, but it hangs in a redis call and I can't figure out why. So in the meantime we'll need to keep using the old version, and be very careful to not accidentally upgrade it somewhere.

(this incompatibility is also why the tests aren't running at the moment)

**Comment by jrs65** [comment]
Deployment steps (notes for @jrs65):
- [ ] Change coco.yaml play to
  - Set the branch to this one
  - Remove the coco_server role
- [ ] Run ansible play
- [ ] Manually restart the coco service on csbfs
- [ ] ?
- [ ] Profit

**Pull request merged**